src/main/avro/com.snowplowanalytics.sauna.observers/AmazonKinesisConfig/avro/1-0-0.avsc

```diff
@@ -0,0 +1,46 @@
+{
+  "namespace": "com.snowplowanalytics.sauna.observers",
+  "name": "AmazonKinesisConfig",
+  "type": "record",
+  "fields": [
+    {
+      "name": "enabled",
+      "type": "boolean"
+    },
+    {
+      "name": "id",
+      "type": "string"
+    },
+    { "name": "parameters",
+      "type": {
+        "name": "AmazonKinesisConfigParameters",
+        "type": "record",
+        "fields": [
+          {
+            "name": "awsRegion",
+            "type": "string",
+            "default": null
+          },
+          {
+            "name": "awsAccessKeyId",
+            "type": "string",
+            "default": null
```

Comment:
I'd try to avoid `null`s at all costs. Unfortunately, avrohugger unlike other avro-caseclass generators doesn't translate union of `null` and `A` to `Option[A]`, which makes possible to pass `null` as in https://github.com/snowplow/sauna/pull/68/files#diff-42812c1bf3789fb57a50dce3e2600191R69

On the other hand - I agree that it is very good idea to have an opportunity to use environment variables to configure AWS services. But I think this should be made a) consistently with other AWS Services (S3 observer); b) as separate feature. I'd like to hear your thoughts.

/cc @alexanderdean